server/cli/config.py

```diff
@@ -0,0 +1,7 @@
+import click
+from server.common.default_config import default_config
+
+
+@click.command(short_help="Print the default cellxgene server configuration")
```

Comment:
"server configuration", or just "configuration"?   Given that it affects front & backend, suggest `Print the default cellxgene configuration`